starboard/doc/evergreen/cobalt_evergreen_overview.md

```diff
@@ -507,15 +506,13 @@ updates in an example 3-slot configuration:
     ├── installation_2 <--(SLOT_2 - contains new Cobalt version)
     │   ├── content
     │   │   ├── fonts <--(`empty` configuration)
-    │   │   ├── (icu) <--(only present when it needs to be updated by Cobalt Update)
     │   │   ├── licenses
     │   │   ├── ssl
     │   ├── lib
     │   │   └── libcobalt.so <--(SLOT_2 version of libcobalt.so)
     │   ├── manifest.fingerprint
     │   └── manifest.json <-- (Evergreen version information of libcobalt.so under SLOT_2)
     ├── installation_store_<APP_KEY>.pb
```

Comment:
nit: change this entry to start with `└──`

Reply:
Thanks. I also see other formatting issues in this diagram but since they're all unrelated to this change, I'd rather keep the diff cleaner and reformat the diagram in a quick follow-up PR.

Reply:
Sgtm